.github/workflows/e2e-tests-scheduled.yaml

```diff
@@ -69,7 +69,8 @@ jobs:
         # - 'platform:el8'
         - 'ubuntu:18.04'
         - 'ubuntu:20.04'
-        - 'ubuntu:22.04'
+        # The scheduled E2E test in the release/1.x branch fails until the changes are merged. Disabling for now.
```

Comment:
You can use https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#excluding-matrix-configurations to remove the release/1.4 - 22.04 combination instead of disabling it entirely.

Reply:
Re-using your TLA... TIL.  :)